card/mod/01_history/set/all/actions.rb

```diff
@@ -0,0 +1,47 @@
+# -*- encoding : utf-8 -*-
```

Comment:
this is not an issue for this PR, but I really want to get rid of this file naming pattern (01_whatever) and start representing dependencies in the module.  It really shouldn't be that hard, because the *all modules are all loaded independently first and then later included.  It's just a matter of resorting them first.

(I started the naming pattern with formats; it's my fault.  I just meant it as a temporary hack, not a core pattern.)